File: prog1.c
#include <stdio.h>

int main()
{
    int bin[51];
    int num;
    int i;
    printf("Enter the decimal value: ");
    scanf("%i",&num);

    for (i=0; num>0; i++)
    {
        bin[i]=num%2;
        num=num/2;
    }
    for (int j=i-1; j>=0; j--)
    {
        printf("%d",bin[j]);
    }
    
    return 0;
}

File: prog1.py
dec=int(input("Enter the decimal value: "))
dec2=dec
bin=0
bin_list=[]

while dec!=1:
    rem=dec%2
    dec=dec//2
    bin_list.append(rem)

print(f'Binary conversion for {dec2} is: 1',end='')

for i in range (len(bin_list),0,-1):
    print(bin_list[i-1],end='')